Cargo.toml

```diff
@@ -4,12 +4,6 @@ version = "0.1.0"
 authors = ["{{authors}}"]
 edition = "2021"
 
-exclude = [
-  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
-  "contract.wasm",
-  "hash.txt",
-]
```

Comment:
Those files are not created anymore since rust-optimizer 0.10.0 from 3 years ago

Reply:
What a coincidence, I was wondering about that this morning before seeing this PR.

Reply:
Relicts from an ancient history before you joined :)